tests/pytests/utils.py

```diff
@@ -62,3 +68,17 @@ def tnm():
     """Returns the name of the calling function, usually name of test case."""
 
     return sys._getframe().f_back.f_code.co_name
+
+
+def run_psi4_cli(inputs, outputs, extra_commands=None, as_binary=None):
+    """
+    Runs Psi4 from the CLI in a subprocess.
+    """
+
+    if extra_commands is None:
+        extra_commands = []
+    psi_runner = os.path.join(os.path.dirname(os.path.abspath(psi4.__file__)), "run_psi4.py")
```

Comment:
`psi4.__file__` is generally going to be `<objdir>/stage/lib/psi4/core.....so`, and there isn't a `run_psi4.py` alongside.

Reply:
`psi4.__file__` is the main python `__init__.py`

Reply:
Gotcha here, fixed it up.